src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -265,6 +265,7 @@
         <Product>$(TestProduct)</Product>
         <Architecture>$(Platform)</Architecture>
         <Configuration>$(ConfigurationGroup)</Configuration>
+        <Configuration Condition="'$(ConfigurationSuffix)' != ''">$(Configuration)-$(ConfigurationSuffix)</Configuration>
```

Comment:
I don't see the point of including a hyphen, making it on the next line, or making it conditional.  If it's unset, it's an empty string, and if someone wants a hyphen they can put the hyphen into their suffix.  

This will work though, so I will also say LGTM.